src/getAvailable.js

```diff
@@ -14,7 +14,7 @@ const getAvailable = async (req, res) => {
     return unauthorized(req, res);
   }
 
-  const datasets = await source.availableDatasets() || [];
+  const [datasets, secondTreeOptions] = await source.availableDatasets() || [];
```

Comment:
In the fallback case, this changes behaviour from `datasets = []` to `datasets = undefined`, which will break the subsequent call to `datasets.map(…)` below and thus throw a server error.